app/models/manageiq/providers/amazon/inventory/persister.rb

```diff
@@ -51,11 +39,9 @@ def strategy
   end
 
   def shared_options
```

Comment:
hm, do we still use these somewhere?

Reply:
Do you mean advanced settings? I don't see all contexts yet, but because it has lowest importance from all definitions, you practically cannot add anything which can be useful because it doesn't have ability to overwrite anything - but this decision is up to you guys, I'm new there :)

Reply:
right, so this one was used to turn targeted refresh on (passing 1 setting to every IC) and to pass settings from advanced settings

Reply:
it is passing 2 settings (some providers 3) to every IC - ```strategy``` and ```targeted``` (at low level priority - when ic defines its own, it overwrites this one)

and settings from advanced settings